uk.co.compucorp.civicrm.hrleaveandabsences/tests/phpunit/CRM/HRLeaveAndAbsences/BAO/LeaveRequestTest.php

```diff
@@ -1690,8 +1690,8 @@ public function testLeaveRequestCanNotBeCreatedWhenTheDatesOverlapMoreThanOneCon
     $periodStartDate1 = date('2016-01-01');
     $periodEndDate1 = date('2016-06-30');
 
-    $periodStartDate2 = date('2016-07-01');
-    $periodEndDate2 = date('2016-12-31');
+    $periodStartDate2 = date('2016-07-02');
```

Comment:
It doesn't make sense to use the date function like this. The first param is the date format, not an actual date. We should either use `CRM_Utils_Date::processDate()` (to make it consistent with the other dates) or just assign the date string to the variable.

Also applies to the other test